app/assets/javascripts/show-hide-content.js

```diff
@@ -0,0 +1,172 @@
+;(function (global) {
```

Comment:
@nickcolley nothing, from the looks of it. The frontend work in this PR is almost entirely @carolinegreen . I would assume that with the removal of govuk_frontend_toolkit from this project, the show-hide-content javascript was simply copied across out of pragmatism.